nf_core/pipeline-template/assets/methods_description_template.yml

```
@@ -0,0 +1,25 @@
+id: "${workflow.manifest.name.replace('/', '-')}-methods-description"
```

As this is in the pipeline template, I think it's safer / easier to just use the jinja templating and hardcode the pipeline name. Rather than rendering dynamically on runtime.

```suggestion
id: "{{ name_noslash }}-methods-description"
```